File: SEproject/Data/Student.cs
using System;
using System.Collections.Generic;

namespace SEproject.Data
{
    public partial class Student
    {
        public string Id { get; set; } = null!;
        public string FirstName { get; set; } = null!;
        public string LastName { get; set; } = null!;
        public string Email { get; set; } = null!;
        public string Major { get; set; } = null!;
    }
}

File: SEproject/ClientApp/src/app/data/models/student.ts
export class Student {
    studentID: string;
    firstName: string;
    lastName: string;
    email: string;
    major: string;

    constructor(id: string, firstName: string, lastName: string, email: string, major: string) {
        this.studentID = id;
        this.firstName = firstName;
        this.lastName = lastName;
        this.email = email;
        this.major = major;
    }
}
